src/app/modules/auth/__tests__/auth.routes.spec.ts

```diff
@@ -71,7 +71,9 @@ describe('auth.routes', () => {
       // Assert
       expect(response.status).toEqual(401)
       expect(response.text).toEqual(
```

Comment:
Can use `response.body` to check instead of `repsonse.text`

Reply:
that's so much better, thanks for this